package.json

```diff
@@ -59,6 +59,7 @@
     "build": "grunt build",
     "release": "grunt release",
     "start": "node scripts/kibana --dev",
+    "repl": "node scripts/kibana --dev --repl --no-base-path",
```

Comment:
Is the REPL considered an alternative way of running Kibana? If not, and it's more of a debugging tool I think we should change the default port Kibana starts on for it. Otherwise, you will need to specify a different port when starting the REPL if you already have Kibana running.

Reply:
Is there a reason for the Kibana server to even listen when using the repl? What if you had to call `server.listen()` or pass `--server.autoListen=true` to trigger auto listening?

Reply:
Personally, I prefer having the repl launch Kibana normally, so that I can just launch the repl and go. I can theoretically also use the REPL to tinker with server-side code, and see the results in my browser. (At least, that's a scenario I've done in Clojure in the past, and found very handy.)